### PR TITLE
Add `name.abbreviated` devPreview app manifest property

### PR DESCRIPTION
**Pull request by erikadoyle**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/32ca1b57256bfa703374251e87563189b2da3446/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 85 | 135 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1cadc240-ae6b-4035-aefa-53262bd49b96) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/32ca1b57256bfa703374251e87563189b2da3446/msteams-platform%2Fwhats-new.md) | 90 | 93 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e5fbde02-dcd9-4a12-a261-c280a4a8a7a9) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/df21327527325d192663237018ad8f704f31e3e8/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 85 | 135 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2a1e9c91-a929-4fc8-9e15-ee173a6280e4) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/df21327527325d192663237018ad8f704f31e3e8/msteams-platform%2Fwhats-new.md) | 90 | 93 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bdcef5dd-5e7b-4e51-8ba5-28aa471aeec4) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/whats-new.md [hunk 1]
```diff
@@ -428,10 +429,9 @@ Discover Microsoft Teams platform features that are in developer preview. You ca
 
 Developer preview is a public program that provides early access to unreleased Teams platform features.
 
-**2024 October**
+**2024 November**
 
-* ***October 17, 2024***: [Specify runtime requirements in app manifest to tailor your app's behavior in Microsoft 365 hosts.](m365-apps/specify-runtime-requirements.md)
-* ***October 17, 2024***: [copilotExtensions renamed to copilotAgents in developer preview app manifest.](./resources/schema/manifest-schema-dev-preview.md#copilotagents)
+* ***November 18, 2024***: [Introduced optional `abbreviated` property for app display name on the app bar.](m365-apps/specify-runtime-requirements.md)
```
Comment (v-kushals) [inline msteams-platform/whats-new.md:434]:
```suggestion
* ***November 18, 2024***: [Introduced optional `abbreviated` property for app display name on the app bar.](./resources/schema/manifest-schema-dev-preview.md#name)
```

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/7a01713a62405558b1f6a4aa675e22244dd0e26e/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 85 | 135 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2c0c97ed-b033-4a40-a7fc-fb9d64aff27c) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/7a01713a62405558b1f6a4aa675e22244dd0e26e/msteams-platform%2Fwhats-new.md) | 90 | 93 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b16214f9-5498-465b-8fc8-912f3e30cc95) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3e306c4c6e2a7b50fb4408479143f722ba5fe3bd/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 85 | 135 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8c5f883d-8c48-4e66-8bb8-aeaf3daffe34) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3e306c4c6e2a7b50fb4408479143f722ba5fe3bd/msteams-platform%2Fwhats-new.md) | 90 | 94 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8aaf7a9f-bccc-4174-ab20-620119801402) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-kushals — APPROVED** [review]

File: msteams-platform/resources/schema/manifest-schema-dev-preview.md [hunk 2]
```diff
@@ -452,6 +452,7 @@ The name of your app experience, displayed to users in the Teams experience. For
 |---|---|---|---|---|
 |`short`|String|30 characters|✔️|The short display name for the app.|
 |`full`|String|100 characters|✔️|The full name of the app. It's used if the full app name exceeds 30 characters.|
+|`abbreviated`|String|15 characters| |Abbreviated name for the app; used as the display name on the app bar. If not specified, `short` name is used on the app bar.|
```
Comment (Wajeed-msft) [inline msteams-platform/resources/schema/manifest-schema-dev-preview.md:455]: Could you please hyperlink where app bar is mentioned in Dev Docs?

**Review by Wajeed-msft — DISMISSED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/7c2452bd6648297d06d7f60160edd406c8ff9d77/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 85 | 135 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6632b620-bc76-4bed-853b-11ba012cfcc2) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/7c2452bd6648297d06d7f60160edd406c8ff9d77/msteams-platform%2Fwhats-new.md) | 90 | 94 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c6f03f7c-5ffc-4b9b-9cde-cff3e1bbb927) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/resources/schema/manifest-schema-dev-preview.md (hunk 2 above)
Comment (surbhigupta12) [reply msteams-platform/resources/schema/manifest-schema-dev-preview.md]: Updated

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**